phys/module_ra_goddard.F

```diff
@@ -2079,7 +2084,7 @@ subroutine goddardrad( sw_or_lw, dx    &
       do kt=kts-1,kte !full level ( kts -1 = TOA, kte = BOA )
         DO ic=1,min(CHUNK,ite-ii+1)
         if(lmask(ic) .eqv. .true.) then
-         flx(ic,kt) = flx(ic,kt) * solcon * cosz(ic)
+         flx(ic,kt) = flx(ic,kt) * solcon * cosz(ic)* ( 1 - obscur(i,j) )! amontornes-bcodina 2015/09 solar eclipses
```

Comment:
@irowebbn @dudhia @weiwangncar 
Folks,
This is an inline computation that cannot be avoided. That means that we cannot packed this variable. Is this an OK way to proceed?

Reply:
This may need some work. Since it is just a 2D variable, not able to package it may not be a big deal. But I am curious if it uses more time if there is no eclipse.